content-src/asrouter/components/SnippetBase/SnippetBase.jsx

```diff
@@ -23,7 +23,7 @@ export class SnippetBase extends React.PureComponent {
       <div className="innerWrapper">
         {props.children}
       </div>
-      <button className="blockButton" onClick={this.onBlockClicked} />
+      <button className="blockButton" title={this.props.content.block_button_text} onClick={this.onBlockClicked} />
```

Comment:
Can we make this to default to `Remove this`?

Reply:
We didn't want to add fallbacks in the templates because right now we don't have an option to provide updates to them. I think it would be best if snippets would provide them until we get https://bugzilla.mozilla.org/show_bug.cgi?id=1433218